### PR TITLE
Update angular dependencies

### PR DESCRIPTION
**Pull request by leoaraujocosta**
• Update angular dependencies

#### Issue: #108

### CODE REVIEW

**Comment by thepercival** [comment]
yes, this would be appreciated

File: package.json [hunk 1]
```diff
@@ -59,7 +59,7 @@
     "url": "https://github.com/nicky-lenaers/ngx-scroll-to/issues"
   },
   "peerDependencies": {
-    "@angular/core": "^6.0.0",
+    "@angular/core": "^7.0.0",
```
Comment (BergerLand) [inline package.json]: I think this should be 4-7 not just ~7

Comment (SmallhillCZ) [reply package.json]: or just

```suggestion
    "@angular/core": ">=4.0.0",
```

**Comment by henrysachs** [comment]
any update on this?

**Comment by henrysachs** [comment]
so lets just merge this guys should we?

**Comment by leoaraujocosta** [comment]
@henrysachs I thiunk yes.

**Comment by henrysachs** [comment]
@nicky-lenaers  what do you think?

**Comment by nicky-lenaers** [comment]
All, I'm almost done upgrading the lib according to the Angular 7 library inclusion. Support will be added soon.

**Comment by nicky-lenaers** [comment]
Closing this in favor of #114 . Had to do some additional refactoring for the whole repository. Thanks for helping out!

**Pull request closed without merging**